pkg/kubelet/oom_watcher_test.go

```diff
@@ -23,7 +23,7 @@ import (
 
 	"k8s.io/client-go/pkg/api/v1"
 	"k8s.io/client-go/tools/record"
-	cadvisortest "k8s.io/kubernetes/pkg/kubelet/cadvisor/testing"
+	cadvisortest "k8s.io/kubernetes/pkg/kubelet/cadvisor/fake"
```

Comment:
nit: s/cadvisortest/cadvisorfake/

Or just drop the alias. This test is tiny...